packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -91,10 +91,10 @@ class FrameFlameChart extends FlameChart<TimelineFrame> {
     ///
     /// Subtract 2 * [sectionLabelOffset] to account for extra space at the
     /// beginning/end of the chart.
-    final double pxPerMicro =
-        (element.clientWidth - 2 * flameChartInset) / frame.duration;
+    final double pxPerMicro = (element.clientWidth - 2 * flameChartInset) /
+        frame.timeRange.duration.inMicroseconds;
 
-    final int frameStartOffset = frame.startTime;
+    final int frameStartOffset = frame.timeRange.start.inMicroseconds;
```

Comment:
making this be just `time` instead of `timeRange` might make this code read better.